src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (const row of rows) {
+  const copy = row.children[1].cloneNode(true);
+
+  row.children[3].after(copy);
+}
```

Comment:
What is 1 or 3?
Let's move these numbers to specific variables, so we could change their values later © Volodymyr Pavlenko))